freedata_server/p2p_connection.py

```diff
@@ -217,51 +217,50 @@ def connected_irs(self, frame):
         self.is_ISS = False
         self.orign = frame["origin"]
         self.destination = frame["destination_crc"]
-
-        if self.socket_command_handler:
-            self.socket_command_handler.socket_respond_connected(self.origin, self.destination, self.bandwidth)
-
+        #If these 2 lines are not here, the receiving station does not reply back with an ACK to a P2P_CONNECTION_CONNECT packet. Is this intentional? Leaving here for testing for now.
         session_open_frame = self.frame_factory.build_p2p_connection_connect_ack(self.destination, self.origin, self.session_id)
```

Comment:
Yes this is making sense to me. Let's use these inline comment feature, as we can mark discussions as solved for example